gdbinit.py

```diff
@@ -158,8 +159,13 @@ def main() -> None:
         pwndbg.profiling.profiler.start()
 
 
-main()
+try:
```

Comment:
```suggestion
# We wrap everything in try/except so that we can exit GDB with an error code
# This is used by tests to check if gdbinit.py failed
try:
```